index.html

```diff
@@ -1,76 +1,639 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>
+        Food Website
+    </title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
+        integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 
 <body>
-    <div class="first_section">
-        <div class="first_section-heading">
-            <h1 class="first_section-h1"><span>Hi,</span>I am Tithi Saha</h1>
-            <h1 class="first_section-h2">FULL STACK DEVELOPER</h1>
+    <section>
+        <nav>
+            <div class="logo">
+                <img src="./image/logo.png">

```

Comment:
add `alt` attribute to provide alternate text for image while it is loading for user